src/lib.rs

```diff
@@ -411,6 +426,32 @@ fn test_report_formatting_supports_threshold() {
     assert_eq!(formatted_report.chars().count(), 1412)
 }
 
+#[test]
+fn test_report_formatting_supports_filtering_out_errors() {
+    let reporter = CrashReporter::with_token(
+        "abc",
+        "org name",
+        "app id",
+        None,
+        None,
+        Some(300),
+        false,
+        false,
+        true,
+    );
+    let report = utils::test_helper::TestHelper::report_from_json(
+        "src/json_parsing/test_fixtures/crashes.json",
+    );
+
+    let mut crash_list_json: serde_json::Value = json!(report.crash_list);
+    let data = crash_list_json.as_object_mut().unwrap();
+    reporter.filter_out_errors(data);
+
+    let value = &mut data["errorGroups"];
+    let all_crashes: &mut Vec<serde_json::Value> = value.as_array_mut().unwrap();
+    assert_eq!(all_crashes.len(), 26);
```

Comment:
Noice, really nice. Is there also a test case that loads the same json, but doesn't filter out errors? I can't find one. If not, could you add one? Then we have the A/B test.

Reply:
done